### DISCUSSION TITLE
Video 189 - versions and errors when declaring a pretrained model

### DISCUSSION THREAD

**Discussion by slawomirwojtas** [mrdbourke/pytorch-deep-learning#735] [General]
I had a quite of an issue when it came to these lines of code:
`
weights = torchvision.models.EfficientNet_B0_Weights.DEFAULT # .DEFAULT = best available weights
model = torchvision.models.efficientnet_b0(weights=weights).to(device)
`

It just didn't work. The versions of torch and torchvisions where:
2.1.0+cu118
0.16.0+cu118

I decided to run your code where there is an update for those so after running it, the code was fine and the versions were:
torch version: 2.1.1+cu121
torchvision version: 0.16.1+cu121

The thing that puzzles me are the versions. In terms of torch they both read 2.1. so it's not clear why one of them doesn't meet the condition:
`assert int(torch.__version__.split(".")[1]) >= 12, "torch version should be 1.12+"` whilst the other does.

And what are the +cu121 suffices?

**Comment by sametucuncu** [comment]
Similar issue on  https://github.com/mrdbourke/pytorch-deep-learning/discussions/693 might solve your problem.
